### PR TITLE
[MCKIN-9476] Add missing report name that was lost during rebase

### PR DESCRIPTION
**Pull request by xitij2000**
Restores a bit of code that was missed during rebase.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @xitij2000! I've created [OSPR-3096](https://openedx.atlassian.net/browse/OSPR-3096) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by xitij2000** [comment]
@xirdneh Could you review this?

File: lms/djangoapps/instructor_task/tasks_helper/grades.py [hunk 1]
```diff
@@ -718,6 +718,7 @@ def generate(cls, _xmodule_instance_args, _entry_id, course_id, task_input, acti
         # problem locations are selected.
         problem_location = re.sub(r'[:/]', '_', problem_locations)[:200]
         csv_name = 'student_state_from_{}'.format(problem_location)
-        upload_csv_to_report_store(rows, csv_name, course_id, start_date)
+        report_name = upload_csv_to_report_store(rows, csv_name, course_id, start_date)
```
Comment (xirdneh) [inline lms/djangoapps/instructor_task/tasks_helper/grades.py]: I don't think `upload_csv_to_report_store` returns the csv name. Seems like it only [emits](https://github.com/edx-solutions/edx-platform/blob/master/lms/djangoapps/instructor_task/tasks_helper/utils.py#L40) a signal which might be used to update the task.
I tried this locally and I got `'report_name': None`.

Comment (xitij2000) [reply lms/djangoapps/instructor_task/tasks_helper/grades.py]: @xirdneh Ah! That might be another [hawthorn thing then|https://github.com/edx/edx-platform/blob/master/lms/djangoapps/instructor_task/tasks_helper/utils.py#L42]. Will update the PR.

Comment (xitij2000) [reply lms/djangoapps/instructor_task/tasks_helper/grades.py]: I've updated the PR.

**Pull request merged**